conans/client/generators/virtualenv.py

```diff
@@ -104,7 +104,7 @@ def _ps1_lines(self, venv_name):
 
     @property
     def content(self):
-        venv_name = os.path.basename(self.conanfile.conanfile_directory)
+        venv_name = os.path.basename(self.conanfile.conanfile_directory or os.getcwd())
```

Comment:
Now if you install a package with reference specifying a virtualenv it won'thave the conanfile_directory, so it uses directly the getcwd dir.

Reply:
Why? Installing a package with reference, the conanfile_directory should be set anyway, shouldn't it? To the cache "export" folder. Another issue is that we might not that folder.